Resources/Textures/_Nuclear14/Structures/Storage/Toolkits/ToolkitRed.rsi/meta.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": 1,
+    "license": "CC-BY-NC-SA-3.0",
+    "copyright": "converted by Marines for use in nuclear14",
```

Comment:
Where from? If this is a MS13 sprite then head to the dmi file you got it from on github, right click the commit number on the right of the file name, copy link, paste it in here and say "taken from commit X, converted by Marines for N14".